### PR TITLE
zh-trans: add docs/getting-started-guides/ubuntu/upgrades.md

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

File: content/zh/docs/getting-started-guides/ubuntu/upgrades.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+    juju upgrade-charm etcd
+
+<!-- This will handle upgrades between minor versions of etcd.
+Instructions on how to upgrade from 2.x to 3.x can be found
+[here](https://github.com/juju-solutions/bundle-canonical-kubernetes/wiki/Etcd-2.3-to-3.x-upgrade)
+in the juju-solutions wiki. -->
+
+命令将会负责 etcd 的次版本升级。
+在[juju 解决方案的 wiki](https://github.com/juju-solutions/bundle-canonical-kubernetes/wiki/Etcd-2.3-to-3.x-upgrade)里
```
Comment (Mr-Linus) [inline content/zh/docs/getting-started-guides/ubuntu/upgrades.md]:
```suggestion
在 [juju 解决方案的 wiki](https://github.com/juju-solutions/bundle-canonical-kubernetes/wiki/Etcd-2.3-to-3.x-upgrade) 里
```

**Comment by chenrui333** [comment]
@Mr-Linus please redo `lgtm`! 😄

**Pull request merged**